exporter.go

```diff
@@ -422,3 +431,21 @@ func (e *Exporter) addOrReuseLayer(image imgutil.Image, layer layers.Layer, prev
 	e.Logger.Debugf("Layer '%s' SHA: %s\n", layer.ID, layer.Digest)
 	return layer.Digest, image.AddLayerWithDiffID(layer.TarPath, layer.Digest)
 }
+
+func (e *Exporter) makeBuildReport(layersDir string) (BuildReport, error) {
+	var out []BOMEntry
+	for _, bp := range e.Buildpacks {
+		var bpBuildReport BuildReport
```

Comment:
We should check the buildpack API and continue to the next buildpack if it is < 0.5. There is an extreme edge case here where an older buildpack may have had a layer with id `build` which is no longer valid. We don't want to try to decode that `layer.toml` file (with name `build.toml`) as a build report. Although it probably wouldn't actually error (?)... its weird and it's safer just to be strict about it.